### PR TITLE
Fix typos in projectforge-business directory

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S "./projectforge-business/src/main/resources/I18nResources_de.properties,./projectforge-application/src/main/resources/i18nKeys.json,./projectforge-wicket/src/main/webapp/scripts/ace,./plugins/org.projectforge.plugins.merlin/src/main/resources/MerlinI18nResources_de.properties,./plugins/org.projectforge.plugins.ihk/src/main/resources/IHKI18nResources_de.properties" -L aci,afterall,alle,als,atleast,atmost,attributen,childs,diagramm,diagramms,feld,fo,gud,ist,methode,nwe,objekt,oder,offen,re-use,re-used,re-using,referenz,ser,sie,tage,thi,titel,uptodate,unter,vor,whit`

### CODE REVIEW

File: projectforge-business/src/main/java/org/projectforge/business/teamcal/servlet/TeamCalResponseServlet.java [hunk 1]
```diff
@@ -179,7 +179,7 @@ protected void doGet(final HttpServletRequest req, final HttpServletResponse res
 
     resp.setContentType("text/html;charset=UTF-8");
     final Map<String, Object> templateData = new HashMap<>();
-    templateData.put("reponse", I18nHelper.getLocalizedMessage("plugins.teamcal.attendee.response." + statusFinal.getKey()));
+    templateData.put("response", I18nHelper.getLocalizedMessage("plugins.teamcal.attendee.response." + statusFinal.getKey()));
     final String content = renderGroovyTemplate("htmlTemplates/teamEventResponse.html", templateData);
```
Comment (luzpaz) [inline projectforge-business/src/main/java/org/projectforge/business/teamcal/servlet/TeamCalResponseServlet.java:183]: source change

File: projectforge-business/src/main/java/org/projectforge/business/teamcal/servlet/TeamCalResponseServlet.java [hunk 2]
```diff
@@ -189,7 +189,7 @@ private void sendNotValidData(HttpServletResponse resp) throws IOException
   {
     resp.setContentType("text/html;charset=UTF-8");
     final Map<String, Object> templateData = new HashMap<>();
-    templateData.put("reponse", I18nHelper.getLocalizedMessage("plugins.teamcal.attendee.response.error"));
+    templateData.put("response", I18nHelper.getLocalizedMessage("plugins.teamcal.attendee.response.error"));
```
Comment (luzpaz) [inline projectforge-business/src/main/java/org/projectforge/business/teamcal/servlet/TeamCalResponseServlet.java:192]: source change

File: projectforge-business/src/main/kotlin/org/projectforge/framework/persistence/history/EntityOpType.kt [hunk 3]
```diff
@@ -45,7 +45,7 @@ enum class EntityOpType {
     MarkAsDeleted,
 
     /**
-     * The Umark deleted.
+     * The Mark undeleted.
      */
     Undelete
```
Comment (luzpaz) [inline projectforge-business/src/main/kotlin/org/projectforge/framework/persistence/history/EntityOpType.kt:50]: Please review

File: projectforge-business/src/main/resources/I18nResources.properties [hunk 4]
```diff
@@ -949,7 +949,7 @@ fibu.auftrag.status.loi=LOI
 fibu.auftrag.status.optional=Optional position
 fibu.auftrag.status.potenzial=Potential
 fibu.auftrag.statusBeschreibung=Description of status
-fibu.auftrag.titel=Title
+fibu.auftrag.title=Title
 fibu.auftrag.title.add=Create new order / tender
 fibu.auftrag.title.edit=Edit order / tender
 fibu.auftrag.title.heading=Order and tender management
```
Comment (luzpaz) [inline projectforge-business/src/main/resources/I18nResources.properties:955]: Please review

File: projectforge-business/src/main/resources/data/init-test-data-orig.xml [hunk 5]
```diff
@@ -5304,7 +5304,7 @@
     <lastUpdate class="sql-timestamp" id="1132">2013-04-07 18:09:35.002</lastUpdate>
     <deleted>false</deleted>
     <id>2</id>
-    <subject>Legal affaires regular meeting</subject>
+    <subject>Legal affairs regular meeting</subject>
```
Comment (luzpaz) [inline projectforge-business/src/main/resources/data/init-test-data-orig.xml:5307]: please review

File: projectforge-business/src/main/resources/data/pfTestdata.sql [hunk 6]
```diff
@@ -630,7 +630,7 @@ VALUES (162, '2013-04-07 18:09:35.002', FALSE, '2013-04-07 18:09:35.002', FALSE,
         NULL, NULL, 15,
         'MINUTES', NULL,
         '2013-03-28 08:00:00',
-        'Legal affaires regular meeting', 157, 'testdata2', 19, NULL, NULL, NULL);
+        'Legal affairs regular meeting', 157, 'testdata2', 19, NULL, NULL, NULL);
 INSERT INTO t_plugin_calendar_event (pk, created, deleted, last_update, all_day, end_date, external_uid, last_email,
```
Comment (luzpaz) [inline projectforge-business/src/main/resources/data/pfTestdata.sql:634]: Please review

File: projectforge-business/src/main/resources/I18nResources.properties [hunk 7]
```diff
@@ -949,7 +949,7 @@ fibu.auftrag.status.loi=LOI
 fibu.auftrag.status.optional=Optional position
 fibu.auftrag.status.potenzial=Potential
 fibu.auftrag.statusBeschreibung=Description of status
-fibu.auftrag.titel=Title
```
Comment (kreinhard) [inline projectforge-business/src/main/resources/I18nResources.properties:952]: This key shouldn't be changed (it's German ;-)

**Review by kreinhard — APPROVED** [review]

**Comment by kreinhard** [comment]
Thanx very much!

**Pull request merged**